File: openapi/code_samples/go/api_v1_organizations_{organization_id}_clients/get.go
response, err := scalekitClient.M2M().ListOrganizationClients(
  ctx,
  "org_123",
  scalekit.ListOrganizationClientsOptions{
    PageSize: 30,
  },
)
if err != nil {
  // handle error
  return
}
_ = response

File: openapi/code_samples/python/api_v1_organizations_{organization_id}_clients/get.py
# List clients for a specific organization
org_id = "org_123"

response = scalekit_client.m2m_client.list_organization_clients(
    organization_id=org_id,
    page_size=30,
)

clients = response.clients
for client in clients:
    print(f"Client ID: {client.id}, Name: {client.name}")
